code/game/atoms.dm

```diff
@@ -550,6 +550,9 @@
 /atom/proc/hit_by_thrown_carbon(mob/living/carbon/human/C, datum/thrownthing/throwingdatum, damage, mob_hurt = FALSE, self_hurt = FALSE)
 	return
 
+/atom/proc/hit_by_thrown_silicon(mob/living/silicon/S, datum/thrownthing/throwingdatum, damage, mob_hurt = FALSE, self_hurt = FALSE)
+	return
+
```

Comment:
Nearly all of these seem to be copies of hit_by_thrown_carbon. Can you generalize most of the hit_by_thrown_carbon into a new proc, maybe like "hit_by_general" idk, that is called by both the silicon and carbon ones?

Reply:
Will get on it tonight, that should be very doable